src/sdk/namespace_fs.js

```diff
@@ -35,6 +35,19 @@ const buffers_pool = new buffer_utils.BuffersPool({
 const XATTR_USER_PREFIX = 'user.';
 // TODO: In order to verify validity add content_md5_mtime as well
 const XATTR_MD5_KEY = XATTR_USER_PREFIX + 'content_md5';
+const XATTR_VERSION_ID = XATTR_USER_PREFIX + 'version_id';
+const XATTR_PREV_VERSION_ID = XATTR_USER_PREFIX + 'prev_version_id';
+//const XATTR_DELETE_MARKER = XATTR_USER_PREFIX + 'delete_marker';
+const hidden_versions_path = '.versions';
+
+const VER_ENABLED = 'ENABLED';
```

Comment:
why not an enum for versionning as well? like you do for copy status. maybe versioning_state?